lib/pmd.js

```diff
@@ -244,6 +244,7 @@ pmd.getArguments = function(process){
  */
 pmd.generateData = function(config){
   var objs = [];
+  var indexData = [];
```

Comment:
Why do we need to include with each file? Is the goal to not require an explicit TOC procedure since each file would contain a index of all files?